### ISSUE TITLE
vapply error on badViews on version 1.34.2

### ISSUE THREAD

**Issue by lcolladotor** [Bioconductor/BiocCheck#184]
Hi,

I just ran into this error:

```R
* Checking biocViews validity...
Error in vapply(badViews, function(view) { : values must be length 1,
 but FUN(X[[1]]) result is length 2
Calls: <Anonymous> -> BiocCheckRun -> checkBiocViews -> vapply
```

That's from https://github.com/lcolladotor/ExampleBiocWorkshop2023/actions/runs/4325676008/jobs/7552141101#step:25:50 with https://github.com/lcolladotor/ExampleBiocWorkshop2023/tree/9da41d1560ed1343b0fa7b880c5c1e94b2342db6.

It uses `BiocCheck` version 1.34.2 https://github.com/lcolladotor/ExampleBiocWorkshop2023/actions/runs/4325676008/jobs/7552141101#step:21:42

Best,
Leo

**Comment by lcolladotor** [comment]
For reference, here are the `biocViews` terms from the `DESCRIPTION` file:
https://github.com/lcolladotor/ExampleBiocWorkshop2023/blob/c61545067b8b3a4c50f2cc7d873a45aa7d1a69b3/DESCRIPTION#L13

**Comment by LiNk-NY** [comment]
Thanks Leo, it looks like that's an error in 

https://github.com/Bioconductor/BiocCheck/blob/c9137fbb39e3aea109e9bec0cecccbaedadf982c/R/checks.R#L284

when there are more than one suggestions, i.e.,  

```r
c("DNASeq", "RNASeq")
```

**Comment by LiNk-NY** [comment]
Fix in dc86767d10a18ee0509e71d439ba073ab7b6137b 
It has also been ported to `RELEASE_3_16`

**Comment by lcolladotor** [comment]
Awesome, thanks Marcel! 🚀
